content/quickstart/aws/_index.md

```diff
@@ -1,94 +1,16 @@
 ---
-title: AWS
-aliases: ["/reference/aws.html"]
+title: Get Started with AWS
+weight: 1
```

Comment:
The ordering of this menu is a little odd:

![image](https://user-images.githubusercontent.com/3953235/59475851-97d66600-8e02-11e9-8fd2-5651a5ffc396.png)

Shouldn't Get Started come at the top, followed by all the clouds in alphabetical order?

Reply:
Yes, it’s supposed to be at the top and it was before I screwed something up squashing/rearranging commits. Will fix.

Reply:
Fixed